### PR TITLE
cumulative updates / fixes on master

### PR DESCRIPTION
**Pull request by sinsinpub**
Will no desc, plz see git commit comment. Free to change or merge this at any time as anyone like.

### CODE REVIEW

File: src/assets/js/WhoCallsTheFleetItemDb.json [hunk 1]
```diff
@@ -138,17 +138,17 @@
...
-{"id":215,"rarity":4,"type":51,"name":{"ja_jp":"Ro.44水上戦闘機bis","ja_kana":"","ja_romaji":"","zh_cn":"Ro.44水上战斗机bis"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":1,"aa":3,"armor":0,"evasion":2,"hit":2,"los":3,"range":"","distance":3},"dismantle":[1,1,0,2],"rankupgradable":"on","craftable":false,"improvable":true,"improvement":[{"upgrade":false,"req":[[[false,false,false,false,false,true,true],[496]]],"resource":[[80,60,-1,280],[8,-1,4,-1,0,0],[-1,-1,-1,-1,0,0],[0,0,0,0,0,0]]}],"time_created":1489771924871,"_id":"fAZlB7RCNRBRRum3","time_modified":1489775806628,"upgrade_from":[164]}
+{"id":215,"rarity":4,"type":51,"name":{"ja_jp":"Ro.44水上戦闘機bis","ja_kana":"","ja_romaji":"","zh_cn":"Ro.44水上战斗机bis"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":1,"aa":3,"armor":0,"evasion":2,"hit":2,"los":3,"range":"","distance":3},"dismantle":[1,1,0,2],"rankupgradable":"on","craftable":false,"improvable":true,"improvement":[{"upgrade":false,"req":[[[false,false,false,false,false,true,true],[496]]],"resource":[[80,60,0,280],[8,14,4,5,26,2],[14,-1,6,-1,164,1],[0,0,0,0,0,0]]}],"time_created":1489771924871,"_id":"fAZlB7RCNRBRRum3","time_modified":1490201028477,"upgrade_from":[164]}
```
Comment (DynamicSTOP) [inline src/assets/js/WhoCallsTheFleetItemDb.json]: `[14,18,6,7,164,1]`

Comment (sinsinpub) [inline src/assets/js/WhoCallsTheFleetItemDb.json]: dont do it at this side, report the issue to src reop. there are other wrong things i know 😂

Comment (DynamicSTOP) [inline src/assets/js/WhoCallsTheFleetItemDb.json]: made an issue there https://github.com/Diablohu/WhoCallsTheFleet/issues/61

**Comment by Tedster59** [comment]
:+1:

**Pull request merged**